blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -188,28 +191,37 @@ const LargePromoItem = ({ customFields, arcSite }) => {
 		? editableContent(content, "description.basic")
 		: {};
 
-	return (
+	const contentHeading = showHeadline ? content?.headlines?.basic : null;
+	const hasAuthors = showByline && content?.credits?.by.length > 0;
+	const imageSearchField = imageOverrideURL ? "imageOverrideURL" : "imageURL";
+
+	return showOverline ||
+		contentHeading ||
+		showImage ||
+		showDescription ||
+		hasAuthors ||
+		showDate ? (
 		<HeadingSection>
 			<Grid as="article" className={BLOCK_CLASS_NAME}>
-				{showImage ? (
-					<MediaItem {...searchableField("imageURL")} suppressContentEditableWarning>
+				{showImage || editableDescription || contentHeading || hasAuthors || showDate ? (
+					<MediaItem {...searchableField(imageSearchField)} suppressContentEditableWarning>
 						<Conditional
 							component={Link}
 							condition={content?.websites?.[arcSite]?.website_url}
 							href={formatURL(content?.websites?.[arcSite]?.website_url)}
 							onClick={registerSuccessEvent}
 							assistiveHidden
 						>
-							{playVideoInPlace ? (
+							{embedMarkup ? (
 								<Video
 									aspectRatio={aspectRatio}
-									embedMarkup={customFields?.content?.embed_html}
+									embedMarkup={embedMarkup}
 									viewportPercentage={viewportPercentage}
 								/>
 							) : (
 								<Image
 									alt={content?.headlines?.basic || null}
-									src={targetImage}
+									src={targetImage || fallbackImage}
```

Comment:
VERY minor change... can we pull this up to line 159:

```
const promoImage = imageOverrideURL || getImageFromANS(content) || fallbackImage;
```
```suggestion
									src={promoImage}
```

Reply:
@nschubach Thanks, added fallbackimage to promoImage[71b773e](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/71b773e11f78a8f2ba8ecd4d58764d3ac2ef3a42)